posthog/hogql_queries/web_analytics/stats_table_pre_aggregated.py

```diff
@@ -70,6 +73,10 @@ def _bounce_rate_query(self) -> ast.SelectQuery:
             ),
         )
 
+        filters = self._get_filters(table_name="web_bounces_combined")
+        if filters:
+            query.where = filters
+
```

Comment:
This and the filter for the path query below are the actual fixes, note the different table_names as they're for inner/outer queries